src/app/component/Klarna/Klarna.component.js

```diff
@@ -80,14 +80,67 @@ export class Klarna extends PureComponent {
             script.parentNode.removeChild(script);
         }
 
+        const klarnaScript = document.createElement('script');
+        klarnaScript.setAttribute('id', KLARNA_SCRIPT_ID);
+        klarnaScript.setAttribute('src', 'https://x.klarnacdn.net/kp/lib/v1/api.js');
+        klarnaScript.async = true;
+        document.head.appendChild(klarnaScript);
+    }
+
+    loadPaymentMethod(method) {
+        this.setState({ isLoading: true });
+        localStorage.setItem('kl_pm', method);
+        this.renderScript();
+        this.setState({ canShowPaymentSelector: false });
```

Comment:
It is not a good practice to do multiple setState's in one function. Move to one setState.